teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
+    super(age, name)
```

Comment:
- [ ] Your `initialize` and `super` methods here are missing the parameter and arguments for the `parent_permission` attribute. Kindly add it.